test/auth.test.js

```diff
@@ -27,6 +39,51 @@ test('Clean status code through auth pipeline', t => {
   })
 })
 
+test('defaultRelation: used when relation not specified', t => {
+  t.plan(4)
+
+  const app = Fastify()
+  app.register(fastifyAuth, { defaultRelation: 'or' })
+    .after(() => {
```

Comment:
after after(), use await.

Reply:
I didn't get it, could you elaborate more?

Reply:
Sorry: instead of `after`, use `await`.

Reply:
done, committed the changes